File: app/src/main/java/com/amarnehsoft/holyquran/db/AppDatabase.java
package com.amarnehsoft.holyquran.db;

import android.arch.persistence.db.SupportSQLiteOpenHelper;
import android.arch.persistence.room.Database;
import android.arch.persistence.room.DatabaseConfiguration;
import android.arch.persistence.room.InvalidationTracker;
import android.arch.persistence.room.RoomDatabase;
import android.support.annotation.NonNull;

import com.amarnehsoft.holyquran.model.Aya;
import com.amarnehsoft.holyquran.model.Surah;

@Database(entities = {Surah.class, Aya.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    abstract SuraDao suraDao();
    abstract AyaDao ayaDao();
}

File: app/src/main/java/com/amarnehsoft/holyquran/db/AppDatabase.kt
package com.amarnehsoft.holyquran.db

import androidx.room.Database
import androidx.room.RoomDatabase


@Database(entities = [SuraEntity::class, AyahEntity::class], version = 2)
abstract class AppDatabase : RoomDatabase() {
    abstract fun suraDao(): SuraDao
    abstract fun ayaDao(): AyaDao
}
